docker-build/baseball.sql

```diff
@@ -0,0 +1,42 @@
+use baseball;
+#select * from game limit 10;
+
+drop table if exists batting_avg_wDATEDIFF ;
+CREATE table batting_avg_wDATEDIFF (
+batter int unsigned not null,
+hit int unsigned not null,
+atbat int unsigned not null,
+local_date datetime not null,
+diff_day int not null,
+game_id int unsigned not null
+) engine=InnoDB default charset latin1;
+
+insert into batting_avg_wDATEDIFF (
+game_id,batter,hit,atbat,local_date,diff_day)
+select g.game_id,bc.batter,bc.hit,bc.atBat,g.local_date,DATEDIFF('2012-07-05 00:00:00' ,g.local_date) as date_diff
```

Comment:
When you call this table later, you never actually use this date_dif column. I think you can get rid of it and save yourself a bunch of lines.